### PR TITLE
Migrate from Travis CI to Github Actions

### PR DESCRIPTION
**Pull request by DreierF**
Addresses issue [TS-25266](https://jira.cqse.eu/browse/TS-25266)

- [ ] Changes are tested adequately
- [ ] Agent's README.md updated in case of user-visible changes
- [ ] CHANGELOG.md updated
- [ ] Present new features in [N&N](https://wiki.cqse.eu/pages/viewpage.action?pageId=689566)
- [ ] [TGA Tutorial](https://docs.teamscale.com/tutorial/setting-up-tga-java) updated
- [ ] [TIA Tutorial](https://docs.teamscale.com/tutorial/tia-java/) updated

Please respect the vote of the [Teamscale bot](https://demo.teamscale.com) or flag irrelevant findings as tolerated or false positives. If you feel that the Teamscale config needs adjustment, please state so in a comment and discuss this with your reviewer.

### CODE REVIEW

**Comment by DreierF** [comment]
The upload step currently fails here as Teamscale can't resolve the provided sha1 hash. I already created a ticket for the fact that this fails with 500 status instead of 404. Should be fine as soon as it is merged to the cqse project

**Comment by karottenreibe** [comment]
@DreierF that's one of the things that we can hopefully also address with teamscale-upload and #162

**Comment by DreierF** [comment]
Oh I already switched to the teamscale-upload tool during the migration ^^

**Comment by karottenreibe** [comment]
xD perfect. Then we only need to teach it how to handle PRs in Github actions properly

**Comment by DreierF** [comment]
The main problem is that Teamscale does not analyze the forked repository. AFICS there is nothing that could be fixed on the teamscale-upload side

**Comment by karottenreibe** [comment]
Ah sorry, didn't catch the DreierF:master :-) makes sense

File: .github/workflows/actions.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    - name: Upload coverage to Teamscale
+      if: always() && github.event_name == 'push'
+      run: |
+        curl -L -o teamscale-upload.zip https://github.com/cqse/teamscale-upload/releases/download/v1.0.0/teamscale-upload-linux.zip
+        unzip teamscale-upload.zip
+        ./teamscale-upload \
+          -s https://demo.teamscale.com/ \
+          -p teamscale-jacoco-agent \
+          -u build \
+          -a ${{ secrets.ACCESS_KEY }} \
```
Comment (mpdeimos) [inline .github/workflows/actions.yml:50]: TEAMSCALE_ACCESS_KEY

**Pull request merged**